lib/mixlib/install/generator/powershell/scripts/install_project.ps1

```diff
@@ -16,11 +16,6 @@ function Install-Project {
   [cmdletbinding(SupportsShouldProcess=$true)]
   param (
     # Project to install
-    # chef - Chef Client
-    # chefdk - Chef Development Kit
-    # angrychef - AngryChef
-    # server and container are not valid windows targets
-    [validateset('chef', 'chefdk', 'angrychef')]
     [string]
     $project = 'chef',
```

Comment:
Not sure I follow @rhass. All line 20 does is set the default value of project to `chef`...like we do in the Bourne version here:
https://github.com/chef/mixlib-install/blob/v1.0.11/lib/mixlib/install/generator/bourne/scripts/script_cli_parameters.sh#L16